RecoMTD/TrackExtender/plugins/TrackExtenderWithMTD.cc

```diff
@@ -90,6 +93,62 @@ namespace {
     double timeChi2;
   };
 
+  class TrackSegments {
+  public:
+    TrackSegments() {
+      nSegment_ = 0;
+      segmentPathOvc_.clear();
+      segmentMom2_.clear();
+    }
+
+    inline size_t addSegment(double tPath, double tMom2) {
+      segmentPathOvc_.emplace_back(tPath * c_inv);
+      segmentMom2_.emplace_back(tMom2);
+      nSegment_++;
+
+      LogTrace("TrackExtenderWithMTD") << "addSegment # " << nSegment_ << " s = " << tPath
+                                       << " p = " << std::sqrt(tMom2);
+
+      return nSegment_;
+    }
+
+    inline const double computeTof(double mass_inv2) {
```

Comment:
```suggestion
    inline double computeTof(double mass_inv2) const {
```
it seems more important here to have the method const, not the return value of a basic type

Reply:
I agree